tests/extmod/ussl_basic.py

```diff
@@ -20,12 +20,13 @@
 # print
 print(repr(ss)[:12])
 
-# setblocking
-try:
-    ss.setblocking(False)
-except NotImplementedError:
-    print('setblocking: NotImplementedError')
-ss.setblocking(True)
+# setblocking() propagates call to the underlying stream object, and
+# io.BytesIO doesn't have setblocking() (in CPython too).
```

Comment:
Can't this test be implemented over loopback sockets? (Which I assume supported by most, if not all, ports supporting `usocket` in one way or another)

Reply:
> Can't this test be implemented over loopback sockets? (Which I assume supported by most, if not all, ports supporting `usocket` in one way or another)

I wouldn't make such assumption. Actually, ussl doesn't have much to do with sockets specifically, it's defined to operate on *streams*. So, bringing in sockets would only tangle the matter, making additional dependencies on sockets behavior (and support for loopback sockets shouldn't be taken for granted, and actually even presence of sockets shouldn't be).